curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/data-analysis-example-b.english.md

```diff
@@ -14,11 +14,11 @@ videoId: 0kJz0q0pvgQ
 
 ```yml
 question:
-  text: Question
+  text: 'What does the 'loc' method allow you to do?
```

Comment:
It might be nice to format this as code. Plus this has missing single quotes that causes issues.

```suggestion
  text: 'What does the <code>loc</code> method allow you to do?'
```